joint_groups:

body_groups:
  flange: iiwa_link_ee
